### DISCUSSION TITLE
Serverless defaulted to Chinese

### DISCUSSION THREAD

**Discussion by malebolgia** [serverless/serverless#11191] [Q&A]
Not sure why it defaulted to Chinese? 

npm install serverless -g

![image](https://user-images.githubusercontent.com/922854/176340154-2bd3a078-4330-4951-8c0f-23805915e68c.png)

Thank you!

**Comment by malebolgia** [comment]
looks like its defaulting to serverless-tencent not sure why?

My temporary solution under bin/serverless.js

```javascript
   case 'serverless-tencent':
      require('../scripts/serverless');
      return;
      require('../lib/cli/run-serverless-tencent')().catch((error) => {
        // Expose eventual resolution error as regular crash, and not unhandled rejection
        process.nextTick(() => {
          throw error;
        });
      });
```

**Reply by mnapoli** [reply]
Hi, this is intentional. Here's what our "Setup guide" mentions (https://www.serverless.com/framework/docs/getting-started):

> Note: users based in China get a setup centered around the chinese [Tencent](https://intl.cloud.tencent.com/) provider. To use AWS instead, set the following environment variable: `SERVERLESS_PLATFORM_VENDOR=aws`.

Hope that helps!

**Reply by malebolgia** [reply]
> Hi, this is intentional. Here's what our "Setup guide" mentions (https://www.serverless.com/framework/docs/getting-started):
> 
> > Note: users based in China get a setup centered around the chinese [Tencent](https://intl.cloud.tencent.com/) provider. To use AWS instead, set the following environment variable: `SERVERLESS_PLATFORM_VENDOR=aws`.
> 
> Hope that helps!

Thank you :)

**Reply by tistisman** [reply]
> Hi, this is intentional. Here's what our "Setup guide" mentions (https://www.serverless.com/framework/docs/getting-started):
> 
> > Note: users based in China get a setup centered around the chinese [Tencent](https://intl.cloud.tencent.com/) provider. To use AWS instead, set the following environment variable: `SERVERLESS_PLATFORM_VENDOR=aws`.
> 
> Hope that helps!

I'm not from china, I'm in SEA and yet the CLI is all in chinese =_=

**Reply by tomgtqq** [reply]
What's this?

Note: users based in China get a setup centered around the chinese Tencent provider. To use AWS instead, set the following environment variable: SERVERLESS_PLATFORM_VENDOR=aws

How can I set the environment variable?

sudo npm install -g serverless --param="SERVERLESS_PLATFORM_VENDOR=aws"

or

export SERVERLESS_PLATFORM_VENDOR=aws

sudo npm install -g serverless

Thanks

**Comment by markjoseultra** [comment]
![image](https://github.com/serverless/serverless/assets/49055288/75523f1a-8a6d-4565-9cd7-d35ce9a1c0ff)

Hi Team

tried 
`npm install -g serverless --param="SERVERLESS_PLATFORM_VENDOR=aws" --param="SLS_GEO_LOCATION=no-cn"`

same issue
